Eastern-Research-Group/mywaterway
# Feature/code smell issues

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3720158

## Main Changes:
* Fixes a bunch of code smells. 

## Steps To Test:
1. Look through the code differences and make sure they make sense.
2. Go through the app and try to test most features.
3. Run cypress tests

## TODO:
- [ ] I tried to fix some of the "empty arrow function" code smells, but I have a feeling my solution won't work. We may need to investigate these further.
